.github/workflows/gh-pages.yml

```diff
@@ -42,7 +41,8 @@ jobs:
           restore-keys: |
             docs_database_and_stim_results_cache
       - name: Generate the documentation
-        run: make html
+        run: |
+          uv run --extra all make html
         working-directory: docs/
```

Comment:
To do: Find the source of this error: 

<img width="790" height="124" alt="Image" src="https://github.com/user-attachments/assets/6d522440-ae49-4150-a31e-bbdd9c66e9d8" />

I believe this is because the `Generate the documentation` step relies on using a working directory for the caching action to reduce the docs build time.